concepticondata/conceptlists/Smeltzer-1995-300.tsv

```diff
@@ -0,0 +1,301 @@
+ID	NUMBER	FRENCH	CONCEPTICON_ID	CONCEPTICON_GLOSS	ENGLISH
+Smeltzer-1995-300-1	1	oeil	1248	EYE	eye
+Smeltzer-1995-300-2	2	tête	1256	HEAD	head
+Smeltzer-1995-300-3	3	dent	1380	TOOTH	tooth
+Smeltzer-1995-300-4	4	langue	1205	TONGUE	tongue
+Smeltzer-1995-300-5	5	nez	1221	NOSE	nose
+Smeltzer-1995-300-6	6	oreille	1247	EAR	ear
+Smeltzer-1995-300-7	7	cou	1333	NECK	neck
+Smeltzer-1995-300-8	8	sein	1402	BREAST	breast
+Smeltzer-1995-300-9	9	ongle	1258	FINGERNAIL	fingernail
+Smeltzer-1995-300-10	10	ventre	1251	BELLY	belly
+Smeltzer-1995-300-11	11	nombril	1838	NAVEL	navel
+Smeltzer-1995-300-12	12	genou	1371	KNEE	knee
+Smeltzer-1995-300-13	13	peau	763	SKIN	skin
+Smeltzer-1995-300-14	14	plume	1201	FEATHER	feather
+Smeltzer-1995-300-15	15	corne	1393	HORN (ANATOMY)	horn
+Smeltzer-1995-300-16	16	queue	1220	TAIL	tail
+Smeltzer-1995-300-17	17	oeuf	744	EGG	egg
+Smeltzer-1995-300-18	18	lait	635	MILK	milk
+Smeltzer-1995-300-19	19	homme	1554	MAN	man
+Smeltzer-1995-300-20	20	femme	962	WOMAN	woman
+Smeltzer-1995-300-21	21	pèr			father?
+Smeltzer-1995-300-22	22	mari	1200	HUSBAND	husband
+Smeltzer-1995-300-23	23	mariée	1266	BRIDE	bride
+Smeltzer-1995-300-24	24	nom	1405	NAME	name
+Smeltzer-1995-300-25	25	nuit	1233	NIGHT	night
+Smeltzer-1995-300-26	26	lune	1313	MOON	moon
+Smeltzer-1995-300-27	27	étoile	1430	STAR	star
+Smeltzer-1995-300-28	28	soleil	1343	SUN	sun
+Smeltzer-1995-300-29	29	nuage	1489	CLOUD	cloud
+Smeltzer-1995-300-30	30	pluie	658	RAIN (PRECIPITATION)	rain
+Smeltzer-1995-300-31	31	terre	1228	EARTH (SOIL)	earth
+Smeltzer-1995-300-32	32	poussière	2	DUST	dust
+Smeltzer-1995-300-33	33	pierre	857	STONE	stone
+Smeltzer-1995-300-34	34	eau	948	WATER	water
+Smeltzer-1995-300-35	35	feu	221	FIRE	fire
+Smeltzer-1995-300-36	36	bois	1803	WOOD	wood
+Smeltzer-1995-300-37	37	fumée	778	SMOKE (EXHAUST)	smoke
+Smeltzer-1995-300-38	38	cendre	646	ASH	ash
+Smeltzer-1995-300-39	39	poulet	1318	CHICKEN	chicken
+Smeltzer-1995-300-40	40	vache	1007	COW	cow
+Smeltzer-1995-300-41	41	chien	2009	DOG	dog
+Smeltzer-1995-300-42	42	éléphant	1290	ELEPHANT	elephant
+Smeltzer-1995-300-43	43	chèvre	1502	GOAT	goat
+Smeltzer-1995-300-44	44	oiseau	937	BIRD	bird
+Smeltzer-1995-300-45	45	serpent	730	SNAKE	snake
+Smeltzer-1995-300-46	46	pou	1392	LOUSE	louse
+Smeltzer-1995-300-47	47	arbre	906	TREE	tree
+Smeltzer-1995-300-48	48	écorce	1204	BARK	bark (of tree)
+Smeltzer-1995-300-49	49	feuille	628	LEAF	leaf
+Smeltzer-1995-300-50	50	racine	670	ROOT	root
+Smeltzer-1995-300-51	51	graisse	323	FAT (ORGANIC SUBSTANCE)	fat, grease
```

Comment:
graisse [french] was mappet to ORGANIC FAT OR OIL in  Grollemund-2015-100-28 but to FAT (ORGANIC SUBSTANCE) in Samarin-1967-100-32. Not sure which one is correct.

Reply:
I guess the reason why Grollemund-2015-100-28 is mapped to ORGANIC FAT OR OIL is because its corresponding English gloss (already given in the source) is 'fat, oil'.
But judging by the wikipedia article, it refers more to fats (https://fr.wikipedia.org/wiki/Graisse).